administrator-guides/omnichannel-integrations/livechat-widget/livechat-enterprise/livechat-queues/README.md

```diff
@@ -0,0 +1,73 @@
+# LiveChat Queues
+
+There are three queue types on Rocket.Chat:
+
+- [Auto Selection [default]](#auto-selection)
```

Comment:
All of these three routing systems are not enterprise features.
We have just one enterprise routing system call by `Load Balancing` which is described here.